content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -412,11 +412,26 @@ A PersistentVolume can be mounted on a host in any way supported by the resource
 
 The access modes are:
 
-* ReadWriteOnce -- the volume can be mounted as read-write by a single node
-* ReadOnlyMany -- the volume can be mounted read-only by many nodes
-* ReadWriteMany -- the volume can be mounted as read-write by many nodes
-* ReadWriteOncePod -- the volume can be mounted as read-write by a single Pod.
-  This is only supported for CSI volumes and Kubernetes version 1.22+.
+`ReadWriteOnce:`
+
+ the volume can be mounted as read-write by a single node. ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node.
+
+`ReadOnlyMany:`
+
+ the volume can be mounted as read-only by many nodes.
+
+`ReadWriteMany:`
+
+ the volume can be mounted as read-write by many nodes.
+
+ `ReadWriteOncePod:`
+
+ the volume can be mounted as read-write by a single Pod. Use ReadWriteOncePod access mode if you want to ensure that only one pod across whole cluster can read that PVC or write to it. This is only supported for CSI volumes and Kubernetes version 1.22+.
```

Comment:
I think you meant:
```suggestion
`ReadWriteOnce`
: the volume can be mounted as read-write by a single node. `ReadWriteOnce` access mode still can allow multiple pods to access the volume when the pods are running on the same node.

`ReadOnlyMany:`
: the volume can be mounted as read-only by many nodes.

`ReadWriteMany`
: the volume can be mounted as read-write by many nodes.

 `ReadWriteOncePod`
: the volume can be mounted as read-write by a single Pod. Use `ReadWriteOncePod` access mode if you want to ensure that only one pod across whole cluster can read that PVC or write to it. This access mode is only supported for CSI volumes, and only when the `ReadWriteOncePod` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled for your cluster.
```

Reply:
@sftim  PTAL!

Reply:
I don't see `:` in the live preview even after this change. Is that expected? If yes, then should we replace `the` with `The` for each sentence.

Reply:
`:` here is Markdown to indicate the definition that follows a term.

I think that capitalisation is a nit. Happy to see this approved.